docs/sources/tempo/metrics/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+aliases:
+- /docs/tempo/latest/metrics-generator/prometheus-compatibility
+title: Prometheus compatibility
+menuTitle: Prometheus compatibility
+description: Learn about using tempo as a Prometheus data source.
+weight: 550
+keywords:
+  - Prometheus
+  - TraceQL
+---
+
+# Prometheus compatibility
+
+{{< docs/experimental product="Prometheus-compatible API"  >}}
+
+This experimental Prometheus-compatible API provides a way to visualize metrics out of traces using TraceQL.
+
+The API will have native support in a future Grafana release, possibly using an internal API, deprecating the need for
+this compatibility layer.
+
+## Configure a Grafana datasource
+
+Once the `local-blocks` processor is enabled in your `metrics-generator`
+configuration, a new Prometheus data source can be added in Grafana to use the
+`query-frontend` at the `/prom` endpoint for this.  Set the data source URL
+parameter to `http://<HOST>:<PORT>/prom`.  For example:
+
+    ```
+    http://query-frontend.trace.svc.cluster.local:3200/prom
+    ```
+
+<p align="center"><img src="datasource.png" alt="Metrics visualization in Grafana "></p>
+
+The datasource test will fail with a `404` status code, but this is accepted
+for the experimental feature.
+
+### Query timeouts
+
+The expensive nature of these queries in different environments can mean long
+query times.  As such, users may need to increase timeouts in various places of
+their system to allow enough time for the data to be returned.
```

Comment:
Is this still correct? When we write, we try not to use passive voice or future tense. 
```suggestion
Because of the expensive nature, these queries can take a long time to run in different systems. 
As such, consider increasing the timeouts in various places of
the system to allow enough time for the data to be returned.
```